wp_api/src/login/url_discovery.rs

```diff
@@ -224,6 +227,36 @@ impl AutoDiscoveryAttemptResult {
             Err(_) => None,
         }
     }
+
+    fn is_local_dev_environment(&self) -> bool {
+        match &self.api_discovery_result {
+            Ok(success) => is_local_dev_environment_url(&success.parsed_site_url),
+            Err(error) => {
+                if let Some(parsed_url) = error.parsed_site_url() {
+                    is_local_dev_environment_url(parsed_url)
+                } else {
+                    false
+                }
+            }
+        }
+    }
+
+    /// Does the site look like a WordPress site?
+    fn is_wordpress_site(&self) -> bool {
```

Comment:
If we are adding this helper, we should use it in `AutoDiscoveryResult::is_wordpress_site` as well.

Reply:
Done in 8cd8bef61d5e4f56188e531e9020e5f32b1f6d7f